CODING_GUIDELINES.md

```diff
@@ -207,7 +207,36 @@ class MyClass
 }
 ```
 
-### 8. Misc.###
+### 8. Header inclusion order.###
+The headers should be placed in the following order:
+ 1. Module header (in .cpp)
+ 2. System/Qt/Boost etc. headers (splitted in subcategories if you have many).
```

Comment:
May one split and order them by module in the following way:
```C++11
#include <QtCore/QDateTime>
#include <QtCore/QList>

#include <QtNetwork/QHostAddress>

#include <QtGui/QIcon>
``` 
for those files where all the headers are whithin Qt4/Qt5 common subset (no QtWidgets includes, for example?) 

BTW, are there any plans to drop support for Qt4?